model_cards/LorenzoDeMattei/GePpeTto.md

```diff
@@ -0,0 +1,133 @@
+---
+language: italian
+---
+
+# GePpeTto GPT2 Model 🇮🇹
+
+Pretrained GPT2 117M model for Italian.
+
+You can find further details in the paper:
+
+Lorenzo De Mattei, Michele Cafagna, Felice Dell’Orletta, Malvina Nissim, Marco Guerini "GePpeTto Carves Italian into a Language Model", arXiv preprint. Pdf available at: https://arxiv.org/abs/2004.14253
+
+## Pretraining Corpus
+
+The pretraining set comprises two main sources. The first one is a dump of Italian Wikipedia (November 2019), 
+consisting of 2.8GB of text. The second one is the ItWac corpus (Baroni et al., 2009), which amounts to 11GB of web
+texts. This collection provides a mix of standard and less standard Italian, on a rather wide chronological span, 
+with older texts than the Wikipedia dump (the latter stretches only to the late 2000s).
+
+## Pretraining details
+
+This model was trained using GPT2's Hugging Face implemenation on 4 NVIDIA Tesla T4 GPU for 620k steps.
+
+Training parameters:
+
+- GPT-2 small configuration
+- vocabulary size: 30k
+- Batch size: 32
+- Block size: 100
+- Adam Optimizer
+- Initial learning rate: 5e-5
+- Warm up steps: 10k
```

Comment:
Do you have a link/screenshot to a Tensorboard or to another experiment tracker for the training?

Reply:
sorry we just logged the loss in a file, we didn't use any visual tracker